WordPress/Classes/Models/Post.m

```diff
@@ -118,27 +158,139 @@ - (BOOL)hasLocalChanges
     if (![self isRevision]) {
         return NO;
     }
-    
+
     if ([super hasLocalChanges]) {
         return YES;
     }
-    
+
     Post *original = (Post *)self.original;
     if (!original) {
         return NO;
     }
-    
+
     if (([self.tags length] != [original.tags length]) && (![self.tags isEqual:original.tags])) {
         return YES;
     }
-    
+
     CLLocationCoordinate2D coord1 = self.geolocation.coordinate;
     CLLocationCoordinate2D coord2 = original.geolocation.coordinate;
     if ((coord1.latitude != coord2.latitude) || (coord1.longitude != coord2.longitude)) {
         return YES;
     }
-    
+
     return NO;
 }
 
+#pragma mark - WPPostContentViewProvider Methods
+
+- (NSString *)authorNameForDisplay
+{
+    return self.author;
+}
+
+- (NSURL *)blogURL
+{
+    return [NSURL URLWithString:self.blog.url];
+}
+
+- (NSString *)blogURLForDisplay
+{
+    return self.blog.displayURL;
+}
+
+- (NSInteger)numberOfComments
+{
+    if (self.commentCount) {
```

Comment:
I think it's safe to just have `return self.commentCount.integerValue`. Worst case scenario, commentCount is nil, and the method ends up returning zero. Nevertheless, *absolutely* no need to change a single bit, just a side comment!

:dancer: